pkg/api/unversioned/well_known_taints.go

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I don't think these things belong in API/unversioned.  It's becoming a dumping ground.  Please create a new package and add these in there (preferably related to scheduling or other terminology).